### ISSUE TITLE
[Feature Request] Negative Trim Values

### ISSUE THREAD

**Issue by julianrinaldi** [PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor#46]
I'm a robot vacuum fanatic, and I have a home with 7 separate areas that are separated by stairs. So I currently have 5 vacuums and plan on getting more so I don't have to move any around. It would be very convenient if I could set a negative trim value in order to change the aspect ratio of the map. It would be even better if I could set a height and have the map centered in whatever height I set so that I can get my maps to match dimensions more easily on my dashboard.

**Comment by PiotrMachowski** [comment]
Do you use it with my [map card](https://github.com/PiotrMachowski/Home-Assistant-Lovelace-Xiaomi-Vacuum-Map-card) or with a different one?

**Comment by julianrinaldi** [comment]
With your map card. Did I miss a setting that’s already there?

**Comment by PiotrMachowski** [comment]
No, but I am rewriting the card, maybe a part of this feature can be added also to it.

**Comment by julianrinaldi** [comment]
It would also be nice to be able to click on the map card when no mode is selected and have it pop up larger like other cards do as well.

**Comment by julianrinaldi** [comment]
![E4C1FDEE-FFC3-41EC-A1CB-BF2476D6D5F4_1_105_c](https://user-images.githubusercontent.com/492435/99864648-98843780-2b72-11eb-9102-33ffc11338d2.jpeg)
You can see how they're not exactly lined up. Being able to have a background image and overlay the map on it like the other feature request asked would be amazing. In that photo, the Living Room and Kitchen are the same "level" but separated by stairs. Being able to have the same background image with the vacuum path overlayed onto it would be very cool.
